waf.tf

```diff
@@ -600,6 +602,26 @@ resource "aws_wafv2_web_acl" "waf" {
       }
     }
   }
+  dynamic "rule" {
+    for_each = length(var.country_count_rules) > 0 ? [1] : [0]
+    content {
+      name     = "country_count_rules"
+      priority = 90
```

Comment:
@DCamma that means we do not count requests that are already blocked, correct? i like it

Reply:
correct